volume_scroller@francislavoie.github.io/extension.js

```diff
@@ -36,49 +32,36 @@ class VolumeScroller
         this.scroll_binding = null;
         this.sink_binding = null;
 
-        this.settings.connect('changed::granularity', setGranularity);
-    }
-
-    enable()
-    {
-        if (this.enabled)
-        {
-            this.disable();
-        }
+        this.settings.connect("changed::granularity", setGranularity);
 
         this.enabled = true;
         this.sink = this.controller.get_default_sink();
 
-        this.scroll_binding = this.panel.connect(
-            'scroll-event',
-            (actor, event) => this._handle_scroll(actor, event));
+        this.scroll_binding = this.panel.connect("scroll-event", (actor, event) =>
+            this._handle_scroll(actor, event)
+        );
 
         this.sink_binding = this.controller.connect(
-            'default-sink-changed',
-            (controller, id) => this._handle_sink_change(controller, id));
+            "default-sink-changed",
+            (controller, id) => this._handle_sink_change(controller, id)
+        );
     }
 
-    disable()
-    {
-        if (this.enabled)
-        {
-            this.enabled = false;
-            this.sink = null;
+    disable() {
```

Comment:
I think we're missing this according to https://gjs.guide/extensions/upgrading/gnome-shell-45.html#extensionutils

```suggestion
    disable() {
        this.settings = null;
```